app/src/org/commcare/android/view/c3/Configuration.java

```diff
@@ -37,40 +38,14 @@ public JSONObject getConfiguration() {
         return mVariables;
     }
 
-
-    /**
-     * Parse given string into double
-     *
-     * @param description Something to identify the kind of value, used to augment any error message.
-     */
-    protected double parseXValue(String value, String description) throws InvalidStateException {
-        if (Graph.TYPE_TIME.equals(mData.getType())) {
-            Date parsed = DateUtils.parseDateTime(value);
-            if (parsed == null) {
-                throw new InvalidStateException("Could not parse date '" + value + "' in " + description);
-            }
-            return parseDouble(String.valueOf(parsed.getTime()), description);
-        }
-
-        return parseDouble(value, description);
-    }
-
     /**
-     * Parse given string into double
+     * Parse given double time value into string acceptable to C3.
      *
-     * @param description Something to identify the kind of value, used to augment any error message.
-     */
-    protected double parseYValue(String value, String description) throws InvalidStateException {
-        return parseDouble(value, description);
-    }
-
-    /**
-     * Parse given string into double
-     *
-     * @param description Something to identify the kind of value, used to augment any error message.
+     * @param days The time, measured in days since the epoch.
      */
-    protected double parseRadiusValue(String value, String description) throws InvalidStateException {
-        return parseDouble(value, description);
+    protected String convertTime(double days) {
+        Date d = new Date((long)(days * 24 * 60 * 60 * 1000));

```

Comment:
nice, didn't know we had that constant lying around (or that it's actually `24 * 60 * 60 * 1000 + 1`...?)